frontends/election-manager/src/utils/election.ts

```diff
@@ -269,23 +274,20 @@ export function generateBlankBallots({
 // overvote is possible. Does not overvote candidate contests where you must select a write-in
 // to overvote. See discussion: https://github.com/votingworks/vxsuite/issues/1711.
 //
-// In cases where it is not possible to overvote a ballot style, returns undefined.
+// In cases where it is not possible to overvote a ballot style, returns null.
 export function generateOvervoteBallot({
   election,
   precinctId,
 }: {
   election: Election;
   precinctId: PrecinctId;
-}): Dictionary<string | VotesDict> | undefined {
+}): Ballot | null {
```

Comment:
I have a slight preference for `null` over `undefined` as an explicitly assigned empty value

Reply:
We've generally preferred `undefined` over `null` in `vxsuite`. If you think it'd be better to prefer `null`, that seems like a good discussion to have. For now, mind leaving "unset" values as `undefined`?

Reply:
Ooh good to know! My preference isn't strong, and I'm happy to switch back to `undefined` for consistency